subsys/zigbee/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ zephyr_library_sources_ifdef(
 	cli/zigbee_cli.c
```

Comment:
`...` in commit 787fefda87d1446cf1774a80224cad588e14e217 is not a valid description. I suggest squashing the first two commits before merge anyway, there's no need to introduce non-building SDK code into the tree.